contracts/Tokens/VRT/VRT.sol

```diff
@@ -1,4 +1,4 @@
-pragma solidity ^0.5.16;
+pragma solidity 0.8.13;
```

Comment:
This contract is not upgradable. I think we should avoid to upgrade any VRT related contract if it's not mandatory for the Diamond architecture.